packages/protocol-contracts/scripts/add-liquidity-uniswap-v2.ts

```diff
@@ -0,0 +1,30 @@
+import { parseEther } from "@ethersproject/units";
+import { getAddress } from "@zetachain/addresses";
+import { ethers } from "hardhat";
+
+import { addZetaEthLiquidityUniV2 } from "../test/uniswapV2.helpers";
+
+async function main() {
+  const [deployer] = await ethers.getSigners();
+
+  const zetaTokenAddr = getAddress("zetaToken");
+  const uniswapRouterV2Address = getAddress("uniswapV2Router02", {
+    customNetworkName: "eth-mainnet",
+    customZetaNetwork: "mainnet",
+  });
+
+  await addZetaEthLiquidityUniV2({
+    ETHToAdd: parseEther("1"),
+    deployer,
+    uniswapRouterV2Address,
+    zetaToAdd: parseEther("0.005"),
+    zetaTokenAddress: zetaTokenAddr,
+  });
```

Comment:
How does this work with different cases? (the pair exists and has a different price ratio, the pair doesn't exist).